.circleci/config.yml

```diff
@@ -82,7 +109,7 @@ jobs:
   netlify_manual:
     docker:
       - image: cimg/node:14.21.3-browsers
-    resource_class: xlarge
+    resource_class: large
```

Comment:
We actually need `xlarge` otherwise preview build fails with out of memory error

Reply:
Yah. When looking at the resources for a manual run, it did go green but it was peaking CPU and nearing peaks for Mem, so since this one is less frequently used it's likely much better to keep it at XLarge. Reverting.